samples/bluetooth/gatt/cts.c

```diff
@@ -110,5 +110,5 @@ void cts_notify(void)
 	}
 
 	ct_update = 0;
-	bt_gatt_notify(NULL, &attrs[3], &ct, sizeof(ct));
+	bt_gatt_notify(NULL, &attrs[1], &ct, sizeof(ct));
```

Comment:
In all other places the new index is `old - 1` but here it's `old - 2`. Is that correct?

Reply:
This was wrong actually, the right index would be 2.